api/hackerProfile.js

```diff
@@ -23,127 +23,39 @@ router.get("/", async (req, res) => {
 });
 
 router.put("/", async (req, res) => {
-  // Get the users current hacker profile
```

Comment:
prob didn't mean for these lines to be deleted right?

Reply:
I think thats mostly merge issues, working through them in a bit